geniza/annotations/admin.py

```diff
@@ -21,7 +22,10 @@ class AnnotationAdmin(admin.ModelAdmin):
 
     def display_content(self, obj):
         """format json content for display in admin"""
-        return mark_safe("<pre>%s</pre>" % pprint.pformat(obj.content, indent=2))
+        # use mark_safe to render <pre>, use escape() to prevent rendering content HTML
+        return mark_safe(
+            "<pre>%s</pre>" % escape(pprint.pformat(obj.content, indent=1))
+        )
```

Comment:
@rlskoeser Not strictly needed... In the pretty-printed dict of the content field, I wanted to see that the transcription lines HTML was being generated correctly. Without `escape()` it was being rendered out because of `mark_safe`—which also looked a bit strange, to have rendered HTML inside of `<pre>`. I think there's not really another place to inspect that HTML besides using browser tools, right? So this might be a useful thing even if not really directly related to this work.

Reply:
yes, I see; I wonder why I didn't notice this before? could it be browser dependent?

does seem useful to have, though